realize.go

```diff
@@ -167,14 +167,14 @@ func main() {
 					&cli.BoolFlag{Name: "no-config", Aliases: []string{"nc"}, Value: false, Usage: "Ignore existing configurations."},
 				},
 				Action: func(p *cli.Context) error {
-					fmt.Println(p.String("path"))
+					fmt.Fprintln(style.Output, p.String("path"))
 					if err := r.Blueprint.Add(p); err != nil {
 						return err
 					}
 					if err := r.Record(r); err != nil {
 						return err
 					}
-					fmt.Println(style.Yellow.Bold("[")+"REALIZE"+style.Yellow.Bold("]"), style.Green.Bold("Your project was successfully added."))
+					fmt.Fprintln(style.Output, style.Yellow.Bold("[")+"REALIZE"+style.Yellow.Bold("]"), style.Green.Bold("Your project was successfully added."))
```

Comment:
Separating the second argument into several arguments might help maintain readability here (and for the rest of this file).

Reply:
Thank you, i am just fixing realize.go and then should be all ok to merge in dev branch